### ISSUE TITLE
Email confirmation for notification subscription

### ISSUE THREAD

**Issue by jedabanat** [Trinkle23897/tuixue.online-visa#69]
Similar to https://github.com/Trinkle23897/tuixue.online-visa/issues/67

I'm trying to subscribe to L1 appointments for the Melbourne consulate. I don't receive an email confirmation for the subscription.

**Comment by Trinkle23897** [comment]
Have you tried another email? I guess maybe it's banned by blacklist... or in spam

**Comment by jedabanat** [comment]
Yep tried three different ones. Added dean@tuixue.online to whitelist too.

**Comment by Trinkle23897** [comment]
I checked and it seems the email notification pipeline is working. Could you please send your email address to trinkle23897@gmail.com so that I can have a check? Thanks!

**Comment by sriharsha1507** [comment]
I can confirm that email confirmation is not working for me either. 
Sending you my name to the given email id :) .. 
Thank you

**Comment by Jilda1987** [comment]
I also send twice and don't get emails confirming

**Comment by Trinkle23897** [comment]
> I also send twice and don't get emails confirming

Gmail sometimes has a very long latency. I'm not sure that's the reason or not because the email service is in China and as we all know, the GFW :(

**Comment by Jilda1987** [comment]
What I can do for confirmation? 
I use two mail
Michaelpaster365@gmail.com 
Vi71@live.com 
Plase confirmation them
